cv32e40s/regress/cv32e40s_ci_check.yaml

```diff
@@ -36,71 +40,64 @@ builds:
 
 tests:
   hello-world:
-    build: uvmt_cv32e40s
+    builds: [ uvmt_cv32e40s ]
```

Comment:
Does our scripts support both `build` and `builds`?

Reply:
Internal scripts work with both, but my changes to cv_regress requires the use of builds.